src/index.ts

```diff
@@ -129,9 +129,9 @@ export const checkCreateNightwatchVersion = async () => {
 
   if (latestVersion && currentVersion && latestVersion !== currentVersion) {
     Logger.error(
-      `New version is available ${colors.red(currentVersion)} -> ${colors.green(
+      `We've updated this onboarding tool. ${colors.red(currentVersion)} -> ${colors.green(
```

Comment:
This first line gives a sense that we have already updated the version for them. Can we change it to something like "A new version of this onboarding tool is available"?

Reply:
It looks good. We have a statement that says to upgrade users need to run the command